agent/azure-monitor-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/implementation/utils/UrlParserTargetTest.java

```diff
@@ -13,145 +13,209 @@ public class UrlParserTargetTest {
 
   @Test
```

Comment:
url could be null or empty? If yes, it would be valuable to add tests for these cases.

Reply:
I added tests for empty and malformed. this method shouldn't be called with null url.